docs/en/api/metrics-query-expression.md

```diff
@@ -178,3 +178,28 @@ relabel(service_percentile{label='0,1,2,3,4'}, label='P50,P75,P90,P95,P99')
 
 ### Result Type
 Follow the input expression.
+
+## AggregateLabel Operation
+AggregateLabel Operation takes an expression and performs aggregate calculation on its labeled value results. It aggregates a group of TIME_SERIES_VALUES into a single TIME_SERIES_VALUES.
```

Comment:
The original data format should be labeled value metric? Please pick the term of type SkyWalking defined